pkg/services/ngalert/api/api_trans_dev.go

```diff
@@ -131,3 +136,140 @@ func (api *API) conditionOldEndpointByID(c *models.ReqContext) response.Response
 
 	return response.JSON(200, evalCond)
 }
+
+// conditionEvalEndpoint handles POST /api/alert-definitions/ruleGroupByOldID.
+func (api *API) ruleGroupByOldID(c *models.ReqContext) response.Response {
+	id := c.ParamsInt64("id")
+	if id == 0 {
+		return response.Error(400, "missing id", nil)
+	}
+
+	save := c.Query("save") == "true"
+
+	getAlert := &models.GetAlertByIdQuery{
+		Id: id,
+	}
+
+	if err := bus.Dispatch(getAlert); err != nil {
```

Comment:
I think that would make sense if this were to stay here, but I don't think it will.

For now this lives as a rest API endpoint so I have something to poke at. But eventually I believe this is part of a database migration (that will need to Delete existing alerts, update dashboards with whatever goes in their place, etc.)